src/main/java/edu/kis/powp/jobs2d/command/gui/CommandManagerWindow.java

```diff
@@ -142,37 +140,9 @@ public CommandManagerWindow(DriverCommandManager commandManager) {
 
 	private void loadCommand() {
 		String rawText = inputCommandField.getText();
-		rawText = rawText.toLowerCase();
-		String [] rawLines = rawText.split("\\n");
-		Pattern linePattern = Pattern.compile("(head|cut)\\(\\-?\\d+\\,\\-?\\d+\\)[\\r\\n]?");
-		List<DriverCommand> commands = new LinkedList<>();
-
-		for(String s : rawLines)
-		{
-			Matcher matcher = linePattern.matcher(s);
-			if(!matcher.matches()) {
-				JOptionPane.showMessageDialog(null, "Incorrect command! Don't use spaces between anything and floating numbers.", "Attention!", JOptionPane.ERROR_MESSAGE);
-				return;
-			}
-
-			DriverCommand command = null;
-
-			int firstParenthesis = s.indexOf("(");
-			int commaPos = s.indexOf(",");
-			int firstValue = Integer.parseInt(s.substring(firstParenthesis + 1, commaPos));
-			int secondValue = Integer.parseInt(s.substring(commaPos + 1, s.length() - 1));
-
-			if(s.startsWith("head")) {
-				command = new SetPositionCommand(firstValue, secondValue);
-			}
-			else if(s.startsWith("cut")) {
-				command = new OperateToCommand(firstValue,secondValue);
-			}
-
-			commands.add(command);
-			commandManager.setCurrentCommand(commands, "Command from code below");
-			updateCurrentCommandField();
-		}
+		CommandParser co = new BasicCommandParser();
```

Comment:
FIX: Interface CommandParser should relieve CommandManagerWindow from dependency to its implementations.  Dependency inversion failed, because implementation should be injected.